frontend/src/GlobalDatasources/GlobalDataSourcesPage/index.jsx

```diff
@@ -200,7 +200,7 @@ export const GlobalDataSourcesPage = ({ darkMode = false, updateSelectedDatasour
           <div className="datasource-search-holder">
             <SearchBox
               dataCy={`home-page`}
-              className="border-0 homepage-search"
+              className="border-0 homepage-search datasources-search-padding"
```

Comment:
Can we use a class name like datasources-search-box?
This way, the same CSS class can be reused for future UI changes as well.

Adding padding to the class-name doesn't looks right.